src/components/item/item.ts

```diff
@@ -276,6 +278,7 @@ import { ItemReorder } from './item-reorder';
   template:
     '<ng-content select="[item-start],[item-left],ion-checkbox:not([item-end]):not([item-right])"></ng-content>' +
     '<div class="item-inner">' +
+      '<ion-reorder *ngIf="_reorderSide == \'start\'" class="reorder-start"></ion-reorder>' +
```

Comment:
we don't need reorder-start, reorder-end and have, `ion-reorder` duplicated, we can just use CSS.

```
order: -1
```

can make the trick